sycl/source/detail/queue_impl.hpp

```diff
@@ -98,6 +103,31 @@ class queue_impl {
         MIsProfilingEnabled(has_property<property::queue::enable_profiling>()),
         MHasDiscardEventsSupport(MDiscardEvents &&
                                  (MHostQueue ? true : MIsInorder)) {
+    // The following commented section provides a guideline on how to use the
```

Comment:
Could you please clarify why we need this commented out code here?

Reply:
This was added as a guideline for future work. It has since been uncommented notifications provided without code location information as it is a frequent request from end users. The ability to separate code location information from the instrumentation allows us to make progress without breaking ABI compatibility. When ABI breakage window is available, the code location information can easily be enabled.

Reply:
No longer commented as current features allow us to separate adding instrumentation features from adding code location information which usually breaks ABI. The instrumentation is now enabled and will augment with code location information in the next ABI breakage window as this is a requested instrumentation feature.